pyproject.toml

```diff
@@ -31,6 +31,7 @@ ignore_patterns = ["venv/*", ".venv/*"]
 addopts = ["-vv"]
 filterwarnings = [
   'error',
-  'ignore:X509Extension support in pyOpenSSL is deprecated. You should use the APIs in cryptography.:DeprecationWarning',
-  'ignore:CSR support in pyOpenSSL is deprecated:DeprecationWarning'
+  'ignore:TLSALPN01Server is deprecated:DeprecationWarning',
+  'ignore:TLSALPN01Response is deprecated:DeprecationWarning',
+  'ignore:TLSALPN01 is deprecated:DeprecationWarning'
```

Comment:
These deprecation warnings are a byproduct of upgrading `acme` to `> 4.1`. As far as I can tell, we are not actually using TLSALPN01 challenges, but the code is written to emit these deprecation warnings regardless. 

All of the unit/integration tests still pass. And I'll verify that the acceptance tests pass in CI